examples/expositional/frameworks/cortexchat/cortex_chat_quickstart.ipynb

```diff
@@ -229,7 +229,7 @@
     "\n",
     "snowpark_session = Session.builder.configs(connection_params).create()\n",
     "\n",
-    "provider = Cortex(snowpark_session, \"llama3.1-8b\")\n",
+    "provider = Cortex(snowpark_session.connection, \"llama3.1-8b\")\n",
```

Comment:
we used to only require the `snowpark_session` here. why did we change this?

Reply:
We always needed connection for providers.